concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1525,6 +1526,31 @@ jobs:
                   echo export "${var_name}"=\"$(eval echo \$${var_name})\"
                 done > config/config.sh
 
+      - task: update-kibana-timezone
```

Comment:
Is there a reason for this to be in this post-deploy job as opposed to in the main cf-deploy job with most of the other post-deploy tasks. The advantage of doing that is that it can be run in parallel with them.

AIUI, this job was only introduced for the healthcheck app because we're not currently deploying it in a zero-downtime manner, and it therefore would have broken the availability tests if it was included in the main cf-deploy job. With that in mind, this job probably doesn't have the best name.